scripts/deploy_assisted_service.sh

```diff
@@ -25,15 +23,14 @@ export REGISTRY_SERVICE_PORT=80
 export REGISTRY_SERVICE_HOST_PORT=5000
 
 
-if [[ "${ENABLE_KUBE_API}" == "true" || "${DEPLOY_TARGET}" == "operator" && -z "${OPENSHIFT_VERSIONS}" ]]; then
-    # Supporting versions >= 4.8 for kube-api
-    OPENSHIFT_VERSIONS=$(cat assisted-service/data/default_ocp_versions.json |
-       jq -rc 'del (.["4.6", "4.7"]) | with_entries(
-           {key: .key, value: {rhcos_image: .value.rhcos_image,
-           rhcos_version: .value.rhcos_version,
-           rhcos_rootfs: .value.rhcos_rootfs}})')
-    json_template=\''%s'\'
-    OPENSHIFT_VERSIONS_CMD="OPENSHIFT_VERSIONS=$(printf "${json_template}" "${OPENSHIFT_VERSIONS}")"
+if [[ "${ENABLE_KUBE_API}" == "true" || "${DEPLOY_TARGET}" == "operator" ]]; then
+    # Only OS_IMAGES list is required in kube-api flow (assisted-service is using defaults if missing)
+    if [ -z "${OPENSHIFT_VERSIONS:-}" ]; then
+        export OPENSHIFT_VERSIONS='{}'
+    fi
+    if [ -z "${RELEASE_IMAGES:-}" ]; then
+        export RELEASE_IMAGES='[]'
+    fi
 fi
```

Comment:
Could this section be deleted entirely?

Reply:
We want to test that dynamic version using ClusterImageSet is working, this is the reason for clearing these.

Reply:
Right, it's for simulating the kube-api/operator scenario, in which only OS_IMAGES var is available.